### ISSUE TITLE
Add DICE

### ISSUE THREAD

**Issue by jshin1394** [google/metrax#64]

**Comment by jshin1394** [comment]
Using keras [implementation](https://www.tensorflow.org/api_docs/python/tf/keras/losses/Dice) as base, add the metric to metrax.

**Comment by jshin1394** [comment]
Thank you so much Mrigank!
